### PR TITLE
Fix connector id

### PR DESCRIPTION
**Pull request by hugjobk**
- Fix connector id: Kafka env part should be lower case

### CODE REVIEW

File: src/connector.rs [hunk 1]
```diff
@@ -31,7 +31,7 @@ impl Connector {
     }
 
     fn id(manifest: &Manifest, config: &Config) -> String {
-        format!("{}-{}-{}-{:?}", manifest.author, manifest.name, manifest.version, config.kafka_env)
+        format!("{}-{}-{}-{}", manifest.author, manifest.name, manifest.version, config.kafka_env)
```
Comment (rukolahasser) [inline src/connector.rs:34]: why not just use as_str()?

Comment (hugjobk) [reply src/connector.rs:34]: Already inplemented Display interface so dont need to convert it to str

Comment (hugjobk) [reply src/connector.rs:34]: It looks more standard

File: src/kafka_utils/topic.rs [hunk 2]
```diff
@@ -89,6 +89,13 @@ impl FromStr for Env {
     }
 }
 
+impl fmt::Display for Env {
+    #[inline]
```
Comment (rukolahasser) [inline src/kafka_utils/topic.rs:93]: Please remove it

File: src/connector.rs (hunk 1 above)
Comment (rukolahasser) [reply src/connector.rs:34]: sounds good

**Review by rukolahasser — APPROVED** [review]

**Pull request merged**